docs/api-docs/store-management/shipping/shipping-provider-api.md

```diff
@@ -100,128 +117,212 @@ In the case of errors, include human-readable error messages in the response pay
 
-```http title="Example carrier connection request with connection settings" lineNumbers
-POST https://example.com/shipping/carrier/connection
+## What's next?
+
+### How your app will be connected to a store
+
+Once a merchant installs your app on their store, merchants and API users can connect your carrier to their store by using the connection options that you provided when you submitted your app.    
+
+#### How merchants will use your app
+A merchant can navigate to the Shipping Manager UI to input carrier connection settings. The UI displays the connection option's `label` that you provided when you submitted your app. The following figure shows how connection options are shown to merchants in the control panel, if you provided a text field for API Key and a checkbox field for enabling expedited shipping:
+
+![Connect Carrier via UI](https://storage.googleapis.com/bigcommerce-production-dev-center/images/connection%20settings.png 'Connection options in Shipping Manager') 
+
+After connecting your carrier, a merchant can define and enable multiple real-time shipping methods for your carrier app in any shipping zone. 
+
+#### How API users will use your app
+API users can connect your carrier to the store by using the [Create a carrier connection](/api-reference/store-management/shipping-api/shipping-carrier/postshippingcarrierconnection) endpoint. In the request, API users will send the carrier ID that you obtained during app setup, as well as values for your app's connection options. Specifically, API users will specify the `code` for each connection option as a property under the `connection` object when they connect to your carrier, for example:    
+
+```json title="Example POST request with X-Auth-Token header" lineNumbers
+POST https://api.bigcommerce.com/stores/{{STORE_HASH}}/v2/shipping/carrier/connection
 X-Auth-Token: {{ACCESS_TOKEN}}
 Content-Type: application/json
 Accept: application/json
 
 {
   "carrier_id": "carrier_33",
   "connection": {
-    "key": "userKey",
-    "account_number": "userAccountNumber"
+    "api_key": "example key",
+    "expedited": "1"
   }
 }
 ```
-&nbsp;
-```http title="Example carrier connection request without connection settings" lineNumbers
-POST https://example.com/shipping/carrier/connection
-X-Auth-Token: {{ACCESS_TOKEN}}
-Content-Type: application/json
-Accept: application/json
 
-{
-  "carrier_id": "carrier_33",
-  "connection": {}
-}
-```
+API users can then define and enable a shipping method for your carrier using the [Create a shipping method](/api-reference/store-management/shipping-api/shipping-method/createashippingmethod) endpoint. They can enable multiple real-time shipping methods for your carrier app in any shipping zone. For more info on how API users will use your carrier, see the [Use a Third-Party Shipping Provider](/api-docs/store-management/shipping/use-shipping-provider) article.

```

Comment:
Yes, it does.

I just added a sentence that states that settings options are accepted in the API (though I left out the `settings` object because I didn't show an example here so may be too much detail)